bigcache.go

```diff
@@ -101,12 +100,13 @@ func newBigCache(ctx context.Context, config Config, clock clock) (*BigCache, er
 
 	if config.CleanWindow > 0 {
 		go func() {
+			logger := newLogger(config.Logger)
```

Comment:
This will solve the data race
```suggestion
		logger := newLogger(config.Logger)
		go func() {
```

Reply:
Nice! anyway, replacing ctx is also a good thing for the future.